src/renderers/webgl/WebGLUtils.js

```diff
@@ -45,6 +45,13 @@ function WebGLUtils( gl, extensions, capabilities ) {
 		if ( p === DepthStencilFormat ) return gl.DEPTH_STENCIL;
 		if ( p === RedFormat ) return gl.RED;
 
+		if ( p === RGBFormat ) {
+
+			console.warn( 'THREE.WebGLRenderer: THREE.RGBFormat has been removed. Please ues THREE.RGBAFormat instead. https://github.com/mrdoob/three.js/pull/23228' );
```

Comment:
spelling: "use"

nit: Use "please" when you are asking for a favor -- not when you are giving instruction.
```
Use THREE.RGBAFormat instead.
```